docs/application/dotnet/guides/nui/text.md

```diff
@@ -342,6 +353,19 @@ The following markup elements are currently supported:
 
 
 
+<a name="decorations"></a>
+
+### Use Markup to represent encoded characters
+
+Since Markup text is not allowed to contain some characters unless they are representing tags such as "<" or ">", to use these characters or represent any unicode character, user should use reserved entities.

```

Comment:
Suggest rephrasing to:

Markup text is not allowed to contain some characters unless they are representing tags such as "<" or ">". To use these characters or represent any unicode character, you must use reserved entities.

Please verify for technical accuracy.